server/utils/nunjucksSetup.ts

```diff
@@ -49,4 +52,6 @@ export default function nunjucksSetup(app: express.Express, path: pathModule.Pla
   njkEnv.addFilter('concatValues', (object: Record<string, unknown>) => {
     return Object.values(object).join(', ')
   })
+
+  return njkEnv
```

Comment:
Is the nunjucks environment only need to cater for the cheerio tests? Any other use?

Reply:
The nunjucks environment is normally attached to the express app on startup. However in the case of cheerio tests, we would like to create and return the nunkucks environment to be able to render nunjucks templates without instantiating an express app